cell_classification/segmentation_data_prep_test.py

```diff
@@ -33,9 +49,165 @@ def test_get_image():
         assert not np.array_equal(CD8_img, CD4_img)
 
 
+def prepare_test_data_folders(
+    num_folders, temp_dir, selected_markers, random=False, scale=1.0
+):
+    data_folders = []
+    for i in range(num_folders):
+        folder = os.path.join(temp_dir, "fov_1" + str(i))
+        os.mkdir(folder)
+        data_folders.append(folder)
+        for marker in selected_markers:
+            if random:
+                img = np.random.rand(256, 256) * scale
+            else:
+                img = np.ones([256, 256])
+            imwrite(
+                os.path.join(temp_dir, "fov_1" + str(i), marker + ".tiff"),
+                img,
+            )
+    return data_folders
+
+
+def test_calculate_normalization_matrix():
+
+    # instantiate data_prep, conversion_matrix and markers
+    data_prep = prep_object()
+    conversion_matrix = prepare_conversion_matrix()
```

Comment:
Now that we don't need the conversion matrix, the code will be easier to understand if you just create the list of markers directly, rather than taking them from a function